modules/homepage/src/main/frontend/src/themePage/Sidebar/sidebarStyle.jsx

```diff
@@ -88,17 +87,18 @@ import {
       }
     },
     logoImage: {
-      width: "30px",
+      width: "100%",
       display: "inline-block",
       maxHeight: "30px",
```

Comment:
This should either be increased or removed to accommodate taller horizontal logos like the one for care4kids. 80px seems to work fine for our current logo shapes.
```suggestion
      maxHeight: "80px",
```